project/Formatting.scala

```diff
@@ -0,0 +1,36 @@
+package com.rbmhtechnology.eventuate
+
+import sbt._
+import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
+import com.typesafe.sbt.SbtScalariform
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+
+object Formatting {
+  lazy val formatSettings = SbtScalariform.scalariformSettings ++ Seq(
+    ScalariformKeys.preferences in Compile  := formattingPreferences,
+    ScalariformKeys.preferences in Test     := formattingPreferences,
+    ScalariformKeys.preferences in MultiJvm := formattingPreferences
+  )
+
+  lazy val docFormatSettings = SbtScalariform.scalariformSettings ++ Seq(
+    ScalariformKeys.preferences in Compile  := docFormattingPreferences,
+    ScalariformKeys.preferences in Test     := docFormattingPreferences,
+    ScalariformKeys.preferences in MultiJvm := docFormattingPreferences
+  )
+
+  def formattingPreferences = {
+    import scalariform.formatter.preferences._
+    FormattingPreferences()
+      .setPreference(RewriteArrowSymbols, true)
+      .setPreference(AlignParameters, true)
+      .setPreference(AlignSingleLineCaseStatements, true)
+  }
+
+  def docFormattingPreferences = {
+    import scalariform.formatter.preferences._
+    FormattingPreferences()
+      .setPreference(RewriteArrowSymbols, false)
+      .setPreference(AlignParameters, true)
+      .setPreference(AlignSingleLineCaseStatements, true)
+  }
+}
```

Comment:
I'd prefer having only `setPreference(AlignSingleLineCaseStatements, true)` for both `formatSettings` and `docFormatSettings`. Especially not happy with the arrow rewrites.